.github/workflows/test.yml

```diff
@@ -89,6 +89,7 @@ jobs:
           - 3.1
           - 3.2
           - 3.3
+          - 3.4.0-preview1
```

Comment:
We don't need to add this preview version, we have a job for `head`, which now is `ruby 3.4.0dev` (https://github.com/sporkmonger/addressable/actions/runs/9173838413/job/25223381402#step:5:25)

Last time that job ran was 4 months ago, so I suspect the `bigdecimal` change had not happen yet as it passed back then (but the logs are now gone)